support/cas-server-support-oauth/src/test/java/org/apereo/cas/support/oauth/web/AbstractOAuth20Tests.java

```diff
@@ -321,6 +321,21 @@ protected static OAuthRegisteredService getRegisteredService(final String servic
         return registeredServiceImpl;
     }
 
+    protected static OAuthRegisteredService getRegisteredService(final String serviceId,
```

Comment:
Clean up `protected static OAuthRegisteredService getRegisteredService(final String serviceId,                                                                 final String secret, final Set<OAuth20GrantTypes> grantTypes) {}` to avoid duplicate code.

Reply:
Right. Hope my future PRs will produce you less work...